js/main.js

```diff
@@ -852,14 +864,14 @@ function getInitialLocale() {
 function main() {
 
     history.pushState = (f => function pushState() {
-        const ret = f.apply(this, arguments);
+        const ret = f.apply(this, arguments); // Fixme: Void function return value is used.
```

Comment:
Fixme: Void function return value is used.

Reply:
It is old code. The warning is from the IDE but seems valid so I added the comment. Better documented than ignored.